app/safe/internal/server/handle/handle.go

```diff
@@ -124,6 +124,15 @@ func InitializeRoutes(source *workloadapi.X509Source) {
 			return
 		}
 
+		if r.Method == http.MethodPost && p == "/sentinel/v1/init-completed" {
```

Comment:
VSecM Sentinel calls this endpoint and sends a bunch of commands for VSecM safe to execute in a batch.